src/index.js

```diff
@@ -1,8 +1,8 @@
 // Helpers
 import 'react-app-polyfill/ie11';
 import 'react-app-polyfill/stable';
-import LogRocket from 'logrocket';
-import setupLogRocketReact from 'logrocket-react';
+// import LogRocket from 'logrocket';
```

Comment:
Add polyfills here.

You will need to add one for the "includes" to support IE 11:
![image](https://user-images.githubusercontent.com/13015201/101486551-ba443380-3954-11eb-9b4f-dce22431450d.png)


Again, have a look in the design system src/www/asssets/js/vendor/polyfills folder as this may have already been done. Then just include it here.